spec/factories/target.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  sequence :title do |n|
+    Faker::Lorem.word + " #{n}"
+  end
+
+  factory :target do
+    title       {generate(:title)}
```

Comment:
I would include the sequence logic right here and avoid the separate method:
```
sequence(:title) { |n| "#{Faker::Lorem.word}_#{n}" }
```
Just a suggestion though :)